io.github.francivaldo4334.Bitsafe_cli.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Bitsafe CLI
+Comment=CLI for comunication with Bitsafe Android application
+Exec=/app/bin/bitsafe.sh
+Icon=/app/package/bitsafe_ic.png
```

Comment:
exec is NOT an absolute path

Icon is always just a name. no path, no extension.

and the icon must be in a standard path, with an exportable name.